content/blog/software-metrics-workshop-report.md

```diff
@@ -0,0 +1,284 @@
+---
+title: "Report from URSSI workshop on software credit, citation, and metrics"
+date: 2019-03-24T15:17:22-07:00
+author: "Karthik Ram"
+---
+
+
+The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software. One particular challenge identified in our first workshop was the lack of consensus on credit for research software. There is a need to gain a deeper understanding of the concept of credit for research software and how, as a community, we could define and measure impact. To explore this topic is greater detail we conducted a [topic focused workshop](https://github.com/si2-urssi/software-credit#software-metrics-credit-and-citation-workshop) on software metrics and credit at the [National Center for Ecological Analysis and Synthesis (NCEAS)](https://www.nceas.ucsb.edu/) in Santa Barbara on January 23-24, 2019.  We invited [19 experts](https://github.com/si2-urssi/software-credit#confirmed-participants) to participate in a series of discussions and deliver short presentations on their organization's efforts in this space. Below is a summary of our discussions from the two days.
+
+**Discussion topics**
+
+We began the workshop with a group discussion on whether we should focus on software credit and incentives in general or focus specifically on machine actionable/automated ways of assessing software leading which lead to credit measures and incentives. 
+
+We focused the discussion on what an ideal credit scenario might look like and how to better reward software through citation. In particular we tried to unpack the intent of citation in the context of research software:
+
+- Do I want to cite software to **provide credit**
+
+- Do I want to cite software **for reproducibility**
+
+- Do I want to cite software to **prove my credibility and respectability** (i.e. show that I understand the area/quality signaling/ paying my dues)
+
+- Do I want to cite software to provide **transparency** of my results ("this is what I did")
+
+**Scope of the Software Institute**
+
+What should a software sustainability institute focus on with respect to software credit? Participants advocated for an institute that would support software at all levels, not just software that has been published or archived. Such an institute should support:
+
+
+
+*   New and experienced package authors
+*   Those that rely on scripts as part of their workflow.
+*   How can researchers who share code (that is not packaged in software) get credit for their work in a way that is analogous to credit for methodology?
```

Comment:
this and the two next items don't really fit the "Such an institute should support:" header - maybe reword?